src/main/java/frc/robot/RobotContainer.java

```diff
@@ -87,29 +90,30 @@ public class RobotContainer {
 
  
   public RobotContainer() {
+    AlignPosition.setPosition(AlignPosition.Manual);
     intakeSubsystem = IntakeSubsystem.getInstance();
     climbingSubsystem = ClimbingSubsystem.getInstance();
-    shootCommand = new ShootCommand();
+    shootCommand = new ShootCommand(operator);
```

Comment:
I think we can remove this, since we construct the ShootCommand when we add it as a default command.

Reply:
removed